tianshou/utils/evaluation_protocol_analysis.py

```diff
@@ -0,0 +1,12 @@
+from tbparse import SummaryReader
```

Comment:
Alternatives to using tbparse are 
- from tensorboard.backend.event_processing import event_accumulator
- convert to csv https://github.com/aai-institute/tianshou/blob/master/examples/mujoco/tools.py (this would allow to extend the custom plotter https://github.com/aai-institute/tianshou/blob/master/examples/mujoco/plotter.py)